lib/rucio/daemons/conveyor/common.py

```diff
@@ -78,8 +78,10 @@ class HeartbeatHandler:
     Simple contextmanager which sets a heartbeat and associated logger on entry and cleans up the heartbeat on exit.
     """
 
-    def __init__(self, executable, logger_prefix=None):
+    def __init__(self, executable, renewal_interval, logger_prefix=None):
```

Comment:
It's very nit-picky, but I would add some documentation for the parameters. Especially what the renewal_interval does.